.github/workflows/release.yml

```diff
@@ -0,0 +1,41 @@
+name: Release
+on:
+  pull_request:
+  release:
+    types:
+      - created
+jobs:
+  tests:
+    # TODO: instead of reusing test.yml, trigger `yarn test` directly
+    uses: ./.github/workflows/test.yml
+    secrets: inherit
+  release:
+    runs-on: ubuntu-latest
+    needs: tests
+    permissions:
+      packages: write
+      contents: read
+    steps:
+      - name: Checkout ⬇️
+        uses: actions/checkout@v3
+
+      - name: Setup node 😻
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: '.node-version'
+          cache: 'yarn'
+          # Configure NPM to point to Github's package registry.
+          registry-url: https://npm.pkg.github.com/
+
+      - name: Install Dependencies 🧶
+        run: yarn install --immutable
+
+      - name: Build EDS 🛠️
+        run: yarn build
+
+      # Publish the package on Github's package registry for internal CZI use. You should still
+      # publish to the npm registry separately from this job.
+      - name: Publish package to the Github registry
+        run: npm publish --tag alpha --foreground-scripts true
```

Comment:
checking my understanding: the `--tag alpha` means every version of eds we publish to the GH registry will be an alpha version, is that right? 

Could you explain why if this is intended? thanks!